content/pt-br/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Comunicação entre contêineres no mesmo pod usando um volume compartilhado
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+Esta página mostra como usar um Volume para se comunicar entre dois Containers rodando
+no mesmo pod. Veja também como permitir que os processos se comuniquem por
+[compartilhamento do processo de namespace](/docs/tasks/configure-pod-container/share-process-namespace/)
+entre os containers.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Criando um pod que executa dois contêineres
+
+Neste exercício, você cria um pod que executa dois contêineres. Os dois recipientes
+compartilham um Volume que eles podem usar para se comunicar. Aqui está o arquivo de configuração
+para a cápsula:
+
+{{< codenew file="pods/two-container-pod.yaml" >}}
+
+No arquivo de configuração, você pode ver que o Pod tem um Volume chamado
+`dados compartilhados`.
```

Comment:
I don't think the volume name should be translated here. We're referring to content in the Pod manifest that was not translated. `shared-data` should be kept as-is.

Reply:
+1

Reply:
I think there was a bit of confusion here. The original sentence was correct and well-translated ("você pode ver que o Pod tem um volume chamado..."), it's just the name of the volume that should have been kept verbatim ("`shared-data`" instead of "`dados compartilhados`").